src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -138,30 +114,32 @@ export default Backbone.Model.extend({
   setDefaultFilterStates() {
     this.set({ filters: {}, states: this.getDefaultSelectedStates() });
   },
-  getSelectedStates() {
-    const availableStateFilterIds = this.getAvailableStates().map('id');
-    const selectedStates = this.getStatesFilters();
-
-    return filter(selectedStates, id => contains(availableStateFilterIds, id)).join() || NIL_UUID;
-  },
   getFiltersState() {
     return {
       filters: this.getFilters(),
       states: this.getStatesFilters(),
       defaultStates: this.getDefaultSelectedStates(),
     };
   },
-  getEntityFilter() {
-    const filtersState = this.getFilters();
-    const clinicianId = this.get('clinicianId');
-    const teamId = this.get('teamId');
-    const noOwner = this.get('noOwner');
-    const selectedStates = this.getSelectedStates();
-    const dateFilter = this.getEntityDateFilter();
-
-    const filters = {
-      'owned-by': dateFilter,
-      'shared-by': dateFilter,
+  formatDateRange(dateType, date, rangeType) {
+    const dateFormat = (dateType === 'due_date') ? 'YYYY-MM-DD' : '';
+    return `${ dayjs(date).startOf(rangeType).format(dateFormat) },${ dayjs(date).endOf(rangeType).format(dateFormat) }`;
+  },
+  getDateRange({ dateType, selectedDate, selectedMonth, selectedWeek, relativeDate }) {
+    if (selectedDate) return this.formatDateRange(dateType, selectedDate, 'day');
+
+    if (selectedMonth) return this.formatDateRange(dateType, selectedMonth, 'month');
+
+    if (selectedWeek) return this.formatDateRange(dateType, selectedWeek, 'week');
+
+    relativeDate = relativeDate || 'thismonth';
+    const { prev, unit } = relativeRanges.get(relativeDate).pick('prev', 'unit');
+    const relativeRange = dayjs().subtract(prev, unit);
+
+    return this.formatDateRange(dateType, relativeRange, unit);
+  },
+  getStaticDateFilter() {
```

Comment:
Changed this as we had a couple of checks for if the state id was `shared-by` or `owned-by` in relation to if a date is set, but really we want to know if it's already hard coded..  so this method is called within the app now and is used to look up the filter if it's not user-defined in `getEntityDateFilter`